src/common/emit-and-fire/index.js

```diff
@@ -7,8 +7,14 @@
 function emitAndFire(widget, eventName, eventArg) {
     const originalEmit = widget.emit;
     let event;
+    //if window.CustomEvent evals true on ie11 but does not support new CustomEvent
     if (window.CustomEvent) {
-        event = new CustomEvent(eventName, { detail: eventArg });
+        try {
```

Comment:
Since we are already checking `window.CustomEvent` why do we need a `try/catch`.